packages/testing/src/execution_testing/test_types/chain_config_types.py

```diff
@@ -14,6 +14,7 @@ class ChainConfigDefaults:
     """
 
     chain_id: int = 1
+    rpc_chain_id: int = 1
```

Comment:
I would remove this internal variable, and use only `chain_id`. We can keep both flags but IMO internally they should resolve to the same variable.